<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgTono</class>
 <widget class="QWidget" name="WidgTono">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>95</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="TonoGroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>95</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>95</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>95</height>
    </size>
   </property>
   <property name="title">
    <string>Tonom√©trie</string>
   </property>
   <widget class="UpSpinBox" name="TOOGSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>45</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>70</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="UpSpinBox" name="TOODSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>70</number>
    </property>
   </widget>
   <widget class="QLabel" name="OGlabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>25</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OG</string>
    </property>
   </widget>
   <widget class="QLabel" name="ODlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OD</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="MethodegroupBox">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>0</y>
     <width>120</width>
     <height>95</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>95</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>95</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>95</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="AirRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Air</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="AplanationRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>37</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Aplanation</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AutreRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>69</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Autre</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpSpinBox</class>
   <extends>QSpinBox</extends>
   <header>upspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
